README.md

```diff
@@ -1,10 +1,10 @@
-# flask-base 
```

Comment:
Sorry about the white space changes, my editor has trimmed them away on save. 
If you think this is not acceptable, I can revert these changes.